### PR TITLE
GH-1323 ADD_MULTICHART_TYPES

### PR DESCRIPTION
**Pull request by mlzummo**
Adds new chart types to MultiChart

Fixes GH-1323

Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

**Comment by mlzummo** [comment]
@GordonSmith @jbrundage Please Review 

A gantt chart type needs to be added for HPCC Charts ... gordon do you have the link to something you did in that area and we can create a new chart type with the correct configs/params set by default. If there are any chart types that i am missing from this let me know

File: src/chart/MultiChart.js [hunk 1]
```diff
@@ -43,6 +43,7 @@
     ].map(function(item) { item.family = "2D"; return item;});
     MultiChart.prototype._NDChartTypes = [
         { id: "COLUMN", display: "Column", widgetClass: "chart_Column" },
+        { id: "BAR", display: "Column", widgetClass: "chart_Bar" },
```
Comment (GordonSmith) [inline src/chart/MultiChart.js]: Display looks wrong?

File: src/chart/MultiChart.js [hunk 2]
```diff
@@ -51,16 +52,22 @@
         { id: "GOOGLE_COLUMN", display: "Column (Google)", widgetClass: "google_Column" },
         { id: "GOOGLE_LINE", display: "Line (Google)", widgetClass: "google_Line" },
         { id: "GOOGLE_SCATTER", display: "Scatter (Google)", widgetClass: "google_Scatter" },
+        { id: "GOOGLE_GANTT", display: "Gantt (Google)", widgetClass: "google_Gantt" },
```
Comment (GordonSmith) [inline src/chart/MultiChart.js]: Gantt does not exist

File: src/chart/MultiChart.js [hunk 3]
```diff
@@ -51,16 +52,22 @@
...
+        { id: "GOOGLE_COMBO", display: "Combo (Google)", widgetClass: "google_Combo" },
         { id: "C3_AREA", display: "Area (C3)", widgetClass: "c3chart_Area" },
         { id: "C3_BAR", display: "Bar (C3)", widgetClass: "c3chart_Bar" },
         { id: "C3_COLUMN", display: "Column (C3)", widgetClass: "c3chart_Column" },
         { id: "C3_LINE", display: "Line (C3)", widgetClass: "c3chart_Line" },
         { id: "C3_SCATTER", display: "Scatter (C3)", widgetClass: "c3chart_Scatter" },
         { id: "C3_STEP", display: "Step (C3)", widgetClass: "c3chart_Step" },
+        { id: "C3_COMBO", display: "COMBO (C3)", widgetClass: "c3chart_Combo" },
```
Comment (GordonSmith) [inline src/chart/MultiChart.js]: Change COMBO to Combo

File: src/chart/MultiChart.js [hunk 4]
```diff
@@ -51,16 +52,22 @@
...
         { id: "AM_AREA", display: "Area (amCharts)", widgetClass: "amchart_Area" },
         { id: "AM_BAR", display: "Bar (amCharts)", widgetClass: "amchart_Bar" },
         { id: "AM_LINE", display: "Line (amCharts)", widgetClass: "amchart_Line" },
         { id: "AM_SCATTER", display: "Scatter (amCharts)", widgetClass: "amchart_Scatter" },
+        { id: "AM_COLUMN", display: "Column (amCharts)", widgetClass: "amchart_Column" },
+        { id: "AM_GANTT", display: "Gantt (amCharts)", widgetClass: "amchart_Gantt" },
+        { id: "AM_COMBO", display: "Combo (amCharts)", widgetClass: "amchart_Combo" },
```
Comment (GordonSmith) [inline src/chart/MultiChart.js]: Combo does not exist

File: src/chart/MultiChart.js (hunk 2 above)
Comment (mlzummo) [inline src/chart/MultiChart.js]: its going to by the time 1.10 is released ... just planning ahead ... ill have it done end of week

File: src/chart/MultiChart.js (hunk 4 above)
Comment (mlzummo) [inline src/chart/MultiChart.js]: its awaiting PR and testing .. would like it in there so DSP can start messing with it

**Comment by mlzummo** [comment]
@GordonSmith @jbrundage please re review made changes and review new comments

**Comment by jbrundage** [comment]
looks good

**Comment by mlzummo** [comment]
@GordonSmith bump

**Comment by GordonSmith** [comment]
Let me know once the dependencies are pulled and I can pull this.

**Comment by mlzummo** [comment]
there is some bugs to work out with doing this yet

**Comment by mlzummo** [comment]
need to add material as well

**Comment by GordonSmith** [comment]
Closing in favor of:  https://github.com/hpcc-systems/Visualization/pull/1361

**Pull request closed without merging**